content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -1,99 +1,190 @@
-======================
-Request a down payment
-======================
+=============
+Down payments
+=============
 
 A down payment is a partial payment made by the buyer when a sales contract is concluded. This
-implies both parties' full commitment (seller and buyer) to honor the contract. With a down payment,
-the buyers show their will to acquire the product and agree to pay the rest later, while the sellers
-are obliged to provide the goods by accepting it.
+implies both parties' (seller and buyer) full commitment to honor the contract.
 
-Sometimes a down payment is required for expensive orders or projects. That way, you can protect
-yourself and make sure that your customer is reliable.
+With a down payment, the buyer shows their will to acquire the product, and provides a portion of
+the total payment, agreeing to pay the remaining amount at a later date. In turn, the seller is
+obliged to provide the goods to the buyer, by accepting the down payment, trusting that the
+remaining amount will eventually be paid.
 
-First time you request a down payment
-=====================================
+Initial down payment request
+============================
 
-When a sales order is confirmed, you then have the possibility to create an invoice. Invoices
-are automatically created in drafts so that you can review them before validation. To create an
-invoice, Odoo Sales offers you 3 options:
+When a sales order is confirmed, the option to create an invoice becomes available, via the
+:guilabel:`Create Invoice` button, located in the upper-left corner of the sales order form. When
+clicked, a :guilabel:`Create invoices` pop-up appears.
 
-- Regular invoice
-- Down payment (percentage)
-- Down payment (fixed amount)
+.. note::
+   Invoices are automatically created in drafts, so they can be reviewed before validation.
+
+On the :guilabel:`Create invoices` pop-up, there are 3 options to choose from in the
+:guilabel:`Create Invoice` field:
+
+- :guilabel:`Regular invoice`
+- :guilabel:`Down payment (percentage)`
+- :guilabel:`Down payment (fixed amount)`
+
+.. note::
+   If :guilabel:`Regular Invoice` is selected, then the other fields disappear, as they only
+   pertain to down payment configurations.
 
 In terms of down payment, it can either be a fixed amount or a percentage of the total amount. The
-first time you request a down payment, you can set a percentage or a fixed amount of your choice,
-and select the right income account and taxes. These settings will be reused for future down
-payments.
+first time a down payment is requested, it can be set to a specific percentage of the total amount,
+or a specific fixed amount.
```

Comment:
The sentences in lines 34-36 seem redundant and can be combined into one clear sentence.